src/qibo/models/variational.py

```diff
@@ -506,7 +511,12 @@ def _loss(params, qaoa, hamiltonian, state):
                 state = hamiltonian.backend.cast(state, copy=True)
             qaoa.set_parameters(params)
             state = qaoa(state)
-            return hamiltonian.expectation(state)
+            if mode is None:
+                return hamiltonian.expectation(state)
+            elif mode == "cvar":
+                return Loss_Utils.cvar(hamiltonian, state)
```

Comment:
Then here you can do:
```suggestion
                from qibo.models.utils import cvar
                
                return cvar(hamiltonian, state)
```